src/helpers/buildWalletSections.tsx

```diff
@@ -133,7 +124,7 @@ const withBriefBalanceSection = (
     {
       type: 'PROFILE_STICKY_HEADER',
       uid: 'assets-profile-header-compact',
-      value: totalValue,
+      value: accountBalance,
       isLoadingUserAssets,
```

Comment:
the profile sticky header probably now wants to rely on `isLoadingBalance` as well - `isLoadingBalance` is being passed in with `accountInfo` but we haven't set up the selector for it in this file yet

Reply:
what do you mean it's getting passed in with `accountInfo`? I don't see that anywhere

Reply:
what I meant is that in `useWalletSectionsData`, the `isLoadingBalance` is passed in (along with `isLoadingUserAssets` in the `accountInfo` object - we're just not using it on the `buildWalletSections` file yet